### ISSUE TITLE
Sonarr searches jackett for wrong season number 

### ISSUE THREAD

**Issue by Ae3NerdGod** [Sonarr/Sonarr#4472]
**Describe the bug**

when i manually search a season of the show `American Dad!` sonarr searches the indexer for one season lesss than the season im searching for, and evaluates all releases as invalid because theyre the wrong season.

**To Reproduce**
add `American Dad!` as a show to a sonarr server with jackett as an indexer.
Manually search for a season. searching season 7, for example, will request season 6 releases from jackett, and provide season 6 results. this is the only show i have with an exclamation mark in the title, no other show seems to have this issue

**Expected behavior**
its obvious that i would like to see season 7 results for a manual search of season 7

**Screenshots**
![192 168 1 7_5907 (nerdcl0ud) - VNC Viewer 5_4_2021 2_45_54 AM](https://user-images.githubusercontent.com/2036505/116968828-41d36b80-ac83-11eb-95b2-09c168d26910.png)

**Platform Information (please complete the following information):**
 - OS: ![Capture](https://user-images.githubusercontent.com/2036505/116968910-63ccee00-ac83-11eb-8b24-bacff0e56223.PNG)
 - Docker: no
 - .net Framework (Windows) or mono (macOS/Linux) (System -> Status): .net 4.8.0
 - Browser and Version (Only needed for UI issues): `Version 90.0.4430.93 (Official Build) (64-bit)`
 - Sonarr Version: 3.0.6.1196
 - Sonarr Branch: master i believe? installer says "main"

**Trace Logs**
[sonarr.debug.txt](https://github.com/Sonarr/Sonarr/files/6419419/sonarr.debug.txt)
[sonarr.trace.txt](https://github.com/Sonarr/Sonarr/files/6419420/sonarr.trace.txt)
[sonarr.txt](https://github.com/Sonarr/Sonarr/files/6419421/sonarr.txt)

**Comment by Thirrian** [comment]
https://wiki.servarr.com/Sonarr_FAQ#How_does_Sonarr_handle_scene_numbering_issues_.28American_Dad.21.2C_etc.29.3F

**Comment by bakerboy448** [comment]
Not a bug.

XEM is in play

**Comment by Ae3NerdGod** [comment]
I don't get it, why is there a whole season missing? Why do we not have a choice of mapping to use; xem or tvdb? Seems like a simple solution. Are we just.. ignoring it? Not fixing it? All due respect that's a terrible "solution"

**Comment by Ae3NerdGod** [comment]
This is indeed a bug, just because you "wontfix" for some stupid reason doesn't mean it's not a bug.

**Comment by bakerboy448** [comment]
This is not a bug; 
http://thexem.de/xem/show/4948
https://wiki.servarr.com/Sonarr_FAQ#how_does_sonarr_handle_scene_numbering_issues

If a certain release group is following TVDB numbering and not Scene numbering and they do not already have a services exception / scene mapping entry, then we just need to know the release name and group to add them to the exception pile.

Please use one of the support channels: [forums](https://forums.sonarr.tv/), [subreddit](https://www.reddit.com/r/sonarr/), [discord ](https://discord.gg/Ex7FmFK), or [IRC ](http://webchat.freenode.net/?channels=#sonarr)for support/questions and for further discussions of this topic to avoid bombarding many people with unnecessary emails.

**Comment by Ae3NerdGod** [comment]
Oh yea, why would a release group follow a site called the FREAKING TV DATABASE. it's the release groups fault, obviously. </Sarcasm>

This is a bug, and you need an option in your ui to resolve it. 

I read the wiki, it's a bug.

You explaining the bug in a wiki, doesn't make it not a bug. 

You acknowledging does not revoke it's bug status.

I get it's doing what it's /programmed/ to do and it's doing what "you" (by "you" I henceforth mean "the sonarr devs") programmed it to do correctly, but "you" have programmed a bug sir. It's a bug. The intended feature does not work as intended. That's a bug. It's a bug, that affects 6 known shows. I understand that's not a lot, and on the grand scheme of things is small, hence you guys putting it on the back burner, but if you continue to deny it is a bug entirely, then I will be looking elsewhere for my automated piracy needs, personally, because you are not the type of people I trust to code. 

I say all this with all due respect; How much respect is due is contingent on your acknowledgment of the reality that this is a bug by the second Definition in webster's dictionary and by common sense.

**Comment by Ae3NerdGod** [comment]
I mean really guys you could just put a checkbox and input field in the interactive search box called "search string override" and boom problem solved. No longer have to manually access my indexer and add the torrents by hand. But nooo this "isn't a bug" so it needs no solution. Be real. Too much bs in the world nowadays.

**Comment by markus101** [comment]
> If a certain release group is following TVDB numbering and not Scene numbering and they do not already have a services exception / scene mapping entry, then we just need to know the release name and group to add them to the exception pile.

This is the answer, Sonarr already has systems in place to deal with release groups (generally p2p groups) that follow the TheTVDB and thus conflict with XEM's mappings. An option to disable XEM mappings doesn't fix anything it just changes which releases would be a problem and that's not something we're interested in supporting.

The latest season for AD is already mapped to allow certain release groups mapping based on TheTVDB numbering, doing the same for previous seasons requires identifying which release groups adhere to TVDB numbering for each season because it has not been consistent for all groups across all seasons. Ultimately the best solution would be a XEMv2 where multiple mappings could co-exist and everyone could openly collaborate, but that's a huge undertaking, so we have an interim solution.

> I mean really guys you could just put a checkbox and input field in the interactive search box called "search string override" and boom problem solved.

This also wouldn't help solve the problem, Sonarr would still need to map releases to episodes and get things wrong depending on which numbering system the release followed.

If you want to be civil and not insult myself, the other Sonarr developers and everyone who has responded here and help map things correctly so it works for you and everyone else, let us know or join us in Discord, but continuing to argue that this is a bug and needs an option isn't going to get anywhere.
